pkg/plugin/restore_pvc_action_plugin.go

```diff
@@ -94,6 +94,25 @@ func (p *NewPVCRestoreItemAction) Execute(input *velero.RestoreItemActionExecute
 			UpdatedItem: item,
 		}, nil
 	}
+
+	ctx := context.Background()
+	veleroNs, exist := os.LookupEnv("VELERO_NAMESPACE")
+	if !exist {
+		errMsg := "Failed to lookup the ENV variable for velero namespace"
+		p.Log.Error(errMsg)
+		return nil, errors.New(errMsg)
+	}
+	restConfig, err := rest.InClusterConfig()
+	if err != nil {
+		p.Log.Errorf("Failed to get the rest config in k8s cluster: %v", err)
+		return nil, errors.WithStack(err)
+	}
+
+	backupdriverClient, err := backupdriverTypedV1.NewForConfig(restConfig)
+	if err != nil {
+		return nil, errors.WithStack(err)
+	}
+
```

Comment:
Is moving this code snippet still necessary for this change?

Reply:
moved back